Misc/NEWS.d/next/Library/2017-10-29-10-37-55.bpo-31608.wkp8Nw.rst

```diff
@@ -0,0 +1,2 @@
+Raise a ``TypeError`` instead of crashing, in specific cases where a
+``collections.deque`` subclass returns a non-deque. Patch by Oren Milman.
```

Comment:
I would mention a copy.

Reply:
I think that mentioning the copy won't help much to a user that isn't familiar with the internal implementation.
If you think that i should be more specific, i could mention that the crashes were in multiplication and concatenation.
What do you think?